test/gui/shared/steps/steps.py

```diff
@@ -723,6 +723,7 @@ def step(context, tabName):
 
 def openSharingDialog(context, resource, itemType='file'):
     resource = getResourcePath(context, resource)
+    waitFor(lambda: os.path.exists(resource), context.userData['maxSyncTimeout'] * 100)
```

Comment:
```suggestion
    waitFor(lambda: os.path.exists(resource), context.userData['maxSyncTimeout'] * 1000)
```

if `waitFor` returns false
 - fail test
 - shown error message